2024Robot_Team3484/src/main/cpp/subsystems/IntakeSubsystem.cpp

```diff
@@ -39,17 +39,21 @@ IntakeSubsystem::IntakeSubsystem(
 
     _target_position = STOW_POSITION;
 
-    _pivot_pid_controller->SetP(PID_CONSTANTS.Kp);
-    _pivot_pid_controller->SetI(PID_CONSTANTS.Ki);
-    _pivot_pid_controller->SetD(PID_CONSTANTS.Kd);
+    _pivot_pid_controller->SetP(pidc.Kp);
+    _pivot_pid_controller->SetI(pidc.Ki);
+    _pivot_pid_controller->SetD(pidc.Kd);
     _pivot_pid_controller->SetIZone(PID_IZ_ZONE);
     _pivot_pid_controller->SetFF(PID_CONSTANTS.Kf);
-    _pivot_pid_controller->SetOutputRange(PID_OUTPUTRANGE_MIN, PID_OUTPUTRANGE_MAX);
+    _pivot_pid_controller->SetOutputRange(pid_output_range_min, pid_output_range_max);
 }
 
 void IntakeSubsystem::Periodic() {
     // Runs every 20ms
 
+    #ifdef EN_DIAGNOSTICS
+        SmartDashboard::GetNumber("Intake Angle (deg)", _pivot_encoder->GetPosition()*360);
```

Comment:
This should be `SmartDashboard::PutNumber( )` not `SmartDashboard::GetNumber( )`. Also I suggest adding a diagnostic for the pivot motor's velocity.